app/assets/stylesheets/govuk-component/govspeak/_example.scss

```diff
@@ -7,9 +7,16 @@
 
 .govuk-govspeak {
   .example {
-    border-left: 10px solid $panel-colour;
-    padding-left: 1.5em;
-    margin: 2.5em 0;
+    border-left: 1em solid $panel-colour;
+    padding: 1em 0 1em 1em;
```

Comment:
I thought we were using those spacing rules but it seems like we're not so I'm 👍  to merge, we can have a conversation about consistency another time.